### PR TITLE
design tweaks

### PR DESCRIPTION
**Pull request by cmuecke**
- add intro & idle animations
- improve avatar hover effect
- replace manual layout switch with media queries
- move modules into separate folder
- push more variables & mixins
- improve property order

@matthistuff  & @alexander-heimbuch review please.

### CODE REVIEW

**Comment by alexander-heimbuch** [comment]
retest this please

File: gulpfile.js [hunk 1]
```diff
@@ -71,13 +71,14 @@ gulp.task('server:signaling', function () {
...
+        watch: 'source',
+        ext: 'js hbs less',
         env: {
             'SIGNALING_SERVER': 'localhost:8080',
             'NODE_ENV': 'development'
-        },
-        ignore: ['client', 'gulpfile.js']
+        }
     }).on('restart', function () {
+        console.log('restart');
```
Comment (alexander-heimbuch) [inline gulpfile.js]: console.log kann noch raus

**Comment by cmuecke** [comment]
retest this please

**Comment by matthistuff** [comment]
:+1:

**Pull request merged**